scripts/run-e2e-config.ts

```diff
@@ -19,6 +19,14 @@ export const angular: Parameters = {
   additionalDeps: ['react', 'react-dom'],
 };
 
+export const angularv7: Parameters = {
+  ...baseAngular,
+  version: 'v7-lts',
+  additionalDeps: [...baseAngular.additionalDeps, 'core-js'],
```

Comment:
I'm wondering why `core-js` is needed, can you give me some hints?

Reply:
It's not "needed", but v7 (didn't checked above) installs core-js v2 which cause some issues with storybook